docs/DeveloperWorkflow/automated_checks.rst

```diff
@@ -1,111 +1,85 @@
 
-* In the case where tests fail, we can debug the problem from going through the console output as displayed here.
+-  In the case where tests fail, we can debug the problem from going
+   through the console output as displayed here.
 
 .. figure:: ../images/build.png
-  :width: 600
-  :align: center
-  :alt: GitHub actions Pull Request Build Console
+   :alt: GitHub actions Pull Request Build Console
+   :align: center
+   :width: 600px
 
-  GitHub actions Pull Request Build Console
+   GitHub actions Pull Request Build Console
+
+\* After analysis of the test failure, you may then try to debug the
+test locally as well by running the command:-:
 
-* After analysis of the test failure, you may then try to debug the test locally as well by running the command:-
 ::
 
-  python -m unittest discover tests -v
+   python -m unittest discover tests -v
 
-* Sometimes tests may also fail if your pull request is not rebased to the latest master. So it is recommended to take a rebase before creating the pull request.
-* If you are still facing issues with the test failure, please reach out to the maintainers of the repository.
+-  Sometimes tests may also fail if your pull request is not rebased to
+   the latest master. So it is recommended to take a rebase before
+   creating the pull request.
+-  If you are still facing issues with the test failure, please reach
+   out to the maintainers of the repository.
 
 Code Coverage Check
 -------------------
 
-moja global uses `Codecov <https://codecov.io/>`_ as a tool for tracking coverage of our application. As mentioned above, the Continuous Integration script uploads the coverage report to Codecov.
-Codecov then compares the coverage percentage to that of our base pull request and asserts if the Code coverage has increased/decreased. After evaluation, the CodeCov bot comments on the pull request with the details of our pull request coverage.
+moja global uses `Codecov`_ as a tool for tracking coverage of our
+application. As mentioned above, the Continuous Integration script
+uploads the coverage report to Codecov. Codecov then compares the
+coverage percentage to that of our base pull request and asserts if the
+Code coverage has increased/decreased. After evaluation, the CodeCov bot
+comments on the pull request with the details of our pull request
+coverage.
 
-If the coverage percentage remains same or higher than before, the check passes. Else if the percentage becomes lower, the check fails.
+If the coverage percentage remains same or higher than before, the check
+passes. Else if the percentage becomes lower, the check fails.
 
-* In order to debug and understand the failure of this check you may click on the **Details** tab of the ``codecov/project`` check under Checks tab.
+-  In order to debug and understand the failure of this check you may
+   click on the **Details** tab of the ``codecov/project`` check under
+   Checks tab.
 
 .. figure:: ../images/all_checks.png
-  :width: 600
```

Comment:
Is this section really not required now?

Reply:
It was missed during formatting. Added it back.